### PR TITLE
Added dns api support for internet.bs

### PR DESCRIPTION
**Pull request by Ne-Lexa**
Added internet.bs DNS API

### CODE REVIEW

**Comment by Ne-Lexa** [comment]
dns_internetbs

File: dnsapi/dns_internetbs.sh [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+  if [ -z "$INTERNETBS_API_KEY" ] || [ -z "$INTERNETBS_API_PASSWORD" ]; then
+    INTERNETBS_API_KEY=""
+    INTERNETBS_API_PASSWORD=""
+    _err "You didn't specify the INTERNET.BS api key and password yet."
+    _err "Please create you key and try again."
+    return 1
+  fi
+
+  _saveaccountconf INTERNETBS_API_KEY "$INTERNETBS_API_KEY"
```
[the remark this answers is not in the document]
Comment (Ne-Lexa) [reply dnsapi/dns_internetbs.sh]: ok

**Comment by Neilpang** [comment]
please fix the conflicts.  and move the doc here:  https://github.com/Neilpang/acme.sh/wiki/dnsapi

Thanks.

**Comment by Ne-Lexa** [comment]
Conflicts have been fixed. Wiki page updated.

**Comment by Neilpang** [comment]
please create a new bug:

https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide#10-please-create-a-new-issue-for-future-bugs

**Comment by Ne-Lexa** [comment]
@Neilpang, ok. 
Issue #2261

**Pull request merged**